frontend/src/profile/GamesTab.tsx

```diff
@@ -20,6 +20,7 @@ interface GamesTabProps {
 }
 
 const GamesTab: React.FC<GamesTabProps> = ({ user }) => {
+    const navigate = useNavigate();
```

Comment:
Can you revert these changes related to useNavigate()? It breaks the headers NextJS has to return to enable Stockfish. Also, we're eventually going to stop using react-router-dom and use the NextJS app router instead

Reply:
What should I do instead? These are existing changes